.github/workflows/ci.yml

```diff
@@ -16,7 +16,11 @@ jobs:
         ruby-version:
           - "3.2"
           - "3.3"
+          - "truffleruby-head"
```

Comment:
We could now add or switch to truffleruby-24.1.2 in CI, e.g. with `"truffleruby"` (latest release), WDYT @fxn ?

Reply:
Right, https://github.com/ruby/setup-ruby/actions/runs/12989568868 was 33min ago and your PR 34min ago so probably just a bit too fast, could you rerun the CI on that PR?

Reply:
Greener! There's just [one failure on Windows](https://github.com/fxn/zeitwerk/actions/runs/12989534412/job/36226275463), error says is is unsupported. Should we remove that pair from the matrix?

Reply:
Yes, the error is accurate, so that combination should be excluded

Reply:
Merged, thanks @eregon!